CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+Unreleased
+-------------
+
+- support mailcow as a mailserver
```

Comment:
```suggestion
- breaking change: mailadm now uses mailcow REST API for creating/manipulating e-mail accounts instead of fiddling with postfix/dovecot config
```